Src/AutoFixture/RandomNumericSequenceGenerator.cs

```diff
@@ -30,8 +30,7 @@ public RandomNumericSequenceGenerator()
         /// <summary>
         /// Initializes a new instance of the <see cref="RandomNumericSequenceGenerator" /> class.
         /// </summary>
-        /// <param name="limits">A sequence of integers beginning with two positive or negative 
-        /// number optionally followed by a series of greater numbers.</param>
+        /// <param name="limits">A sequence of at least two ascending numbers.</param>

```

Comment:
Good – I think, this description is easier to understand than the original one, and it's also more precise.